Fixed missing file location

kzscisoft opened the pull request:
Repository currently refers to `Dropbox` as the root location for the data files even though these files exist within the repository itself. I have switched this to `.` the current directory.

kzscisoft committed:
Removed Dropbox Dependency to instead use data file local to the repository itself

richardreeve commented:
Great. I think there are a couple of other places worth looking for this, e.g.:

https://github.com/ScottishCovidResponse/covid-uk/blob/ff2504244193f878308fecd04d885c4bfd228960/UK.R#L14

It might be worth tracking these down everywhere?

richardreeve commented:
You also might want to change how paths are constructed in R so we're not using things like this:

https://github.com/ScottishCovidResponse/covid-uk/blob/fd1be700a1afe79087851d12a986c0bc7114af3e/UK.R#L15

and instead:

```R
file.path("usr", "local", "lib")
```

Sorry you're starting with the R that you don't know!

kzscisoft committed:
Created Dockerfile to run simulation in container

kzscisoft committed:
Fixed typo

kzscisoft commented:
@richardreeve got to start somewhere! I have added a Dockerfile to the repository now too so the simulation can be run 'out the box' within a container, updated the `README.md` accordingly. I will look into these changes you have suggested now.

kzscisoft committed:
Swapped '.' to 'getwd' and used better path assembly

kzscisoft committed:
Removed irrelevant specific user environment reference

kzscisoft committed:
Updated all path statements in UK*.R

kzscisoft committed:
Fixed file location issue

kzscisoft committed:
Updated README

kzscisoft committed:
'Cherry Picked' changes from source master from 7th May

kzscisoft committed:
Fixed missing bracket

kzscisoft committed:
Added in modifications to R scripts

kzscisoft committed:
Merge branch 'cp-may7th-src'

kzscisoft committed:
fixed commented line

kzscisoft committed:
Updated printed file address

kzscisoft committed:
Updated file paths in R script
